plugins/com.google.cloud.tools.eclipse.appengine.login/java-templates/com/google/cloud/tools/eclipse/appengine/login/Constants.java

```diff
@@ -0,0 +1,10 @@
+package com.google.cloud.tools.eclipse.appengine.login;
+
+/**
+ * Placeholder constants initialized at compile-time.
+ */
+public class Constants {
+
+  public static final String OAUTH_CLIENT_ID = "@oauth.client.id@";
```

Comment:
How are these set? Where do the values actually come from?
Please make sure our docs are updated to include this information.